tools/setup_test.rs

```diff
@@ -63,35 +63,35 @@ fn dl_file(url: &str, to_file: &str, max_size: u64) -> Result<(), DlFileError> {
 
 fn main() -> Result<(), std::io::Error> {
 
-    println!("Using tag: {} for release of massa-unit-tests-src repo...", TAG);
```

Comment:
Those println! are in my opinion quite important to avoid mistakes so maybe we can keep them?

Reply:
Yes my bad this was not supposed to be in the final PR, just some tmp changes that I forgot to revert